hw/ip/prim/prim.core

```diff
@@ -15,6 +15,7 @@ filesets:
       - lowrisc:prim:pad_wrapper
       - lowrisc:prim:prim_pkg
       - lowrisc:prim:clock_mux2
+      - lowrisc:prim:secded
```

Comment:
```suggestion
      - lowrisc:prim:secded # for prim_ram_2p_adv
```

(And potentially push this one up to the top of the list.)

Reply:
Done in latest patchset